_submissions/round-12/1/git-conflict.md

```diff
@@ -0,0 +1,20 @@
+---
+date: 2015-02-23
+round: Round 12
+title: Git Conflict - Auriel Fournier
```

Comment:
Since you're listed as author in the `author` field, we probably don't need it in the `title` - I'll fix it on my end.